### ISSUE TITLE
False report of E121

### ISSUE THREAD

**Issue by gugu** [PyCQA/pycodestyle#92]
This code throws E121 on line 7

``` python
fooff(aaaa,
      abbb,
      cca(
          vvv,
          aaa,
          dadd),
      "here is error: continuation line indentation is not a multiple of four",)
```

But it is just an arguments. When I change code to

``` python
fooff(aaaa,
      abbb,
      cca(vvv,
          aaa,
          dadd),
      "continuation line indentation is not a multiple of four",)
```

PEP8 does not show any error
